### ISSUE TITLE
Make `geoip_detect2_shortcode_get_property` more general

### ISSUE THREAD

**Issue by benjaminpick** [yellowtree/geoip-detect#84]
There are use cases that need more than 2/3 depth - when using ipstack, the data is in extra.original. ...

https://symfony.com/doc/current/components/property_access.html

```php
$propertyAccessor = PropertyAccess::createPropertyAccessorBuilder()->enableMagicCall();

$propertyName = toCamelCase($propertyName);
try {
$propertyValue = $propertyAccessor->getValue($object, $propertyName);
} catch(\Exception $e) {
$propertyValue = '';
}
```

**Comment by benjaminpick** [comment]
Ref: https://wordpress.org/support/topic/zip-code-using-shortcode/

**Comment by benjaminpick** [comment]
This works now BUT this raises the bar to PHP 7.1 . I'm not sure I want to do that.

**Comment by benjaminpick** [comment]
Downgrade property Access to 3.4 - EOL is November 2021

**Comment by benjaminpick** [comment]
Released in 3.1.0
